website/docs/docs/cloud/about-cloud/architecture.md

```diff
@@ -3,52 +3,42 @@ title: Architecture
-- **dbt develop**: This is a server capable of serving dbt IDE requests for a single user. dbt Cloud will create one of these for each user that is actively using the dbt IDE.
+## Communication
 
-#### Application Critical Components
+dbt Cloud can communicate with several external services, including data platforms, git repositories, authentication services, and directories. All communications occur over HTTPS (attempts to connect via HTTP are redirected to HTTPS). dbt Cloud encrypts in transit using the TLS 1.2 cryptographic protocol. 
 
-In addition to the application components, there are a few critical dependencies of the application components that are required in order for the dbt Cloud application to function.
+For more detailed information on our security practices, read our [Security page](https://getdbt.com/security).
 
-- **PostgreSQL database**: dbt Cloud uses a PostgreSQL database as its backend. This can be a cloud-hosted database, for example, AWS RDS, Azure Database, Google Cloud SQL (recommended for production deployments); or, it can be embedded into the dbt Cloud Kubernetes appliance (not recommended for production deployments).
-- **Object Storage**: dbt Cloud requires an S3-compatible Object Storage system for persisting run logs and artifacts.
-- **Storage Volumes**: dbt Cloud requires a Kubernetes storage provider capable of creating dynamic persistent volumes that can be mounted to multiple containers in R/W mode.
+### Data warehouse interaction
 
-### Data Warehouse Interaction
+dbt Cloud's primary role is as a data processor, not a data store. The dbt Cloud application enables users to dispatch SQL to the warehouse for transformation. However, users can post SQL that returns customer data into the dbt Cloud application. This data never persists and will only exist in memory on the instance for the duration of the session. To properly lock down customer data, proper <Term id="data-warehouse" /> permissions must be applied to prevent improper access or storage of sensitive data.
 
-dbt Cloud's primary role is as a data processor, not a data store. The dbt Cloud application enables users to dispatch SQL to the warehouse for transformation purposes. However, it is possible for users to dispatch SQL that returns customer data into the dbt Cloud application. This data is never persisted and will only exist in memory on the instance in question. In order to properly lock down customer data, it is critical that proper <Term id="data-warehouse" /> permissioning is applied to prevent improper access or storage of sensitive data.
+### Git sync
 
-### Deployment Architecture
+dbt Cloud can sync with a variety of git repos, including [Github](/docs/cloud/git/connect-github) and [Gitlab](/docs/cloud/git/connect-gitlab), within it's integrated developement environment ([IDE](/docs/get-started/develop-in-the-cloud)). Communication takes place over HTTPS, rather than SSH, and is protected using the TLS 1.2 protocol for data in transit.
```

Comment:
```suggestion
dbt Cloud can sync with a variety of git repos, including [Github](/docs/cloud/git/connect-github) and [Gitlab](/docs/cloud/git/connect-gitlab), within its integrated development environment ([IDE](/docs/get-started/develop-in-the-cloud)). Communication takes place over HTTPS, rather than SSH, and is protected using the TLS 1.2 protocol for data in transit.
```